ql/src/test/org/apache/hadoop/hive/ql/txn/compactor/TestInitiator.java

```diff
@@ -865,7 +865,7 @@ public void processCompactionCandidatesInParallel() throws Exception {
   }
 
   @Test
-  public void testInitiatorMetricsEnabled() throws Exception {
+  public void testAcidMetricsEnabled() throws Exception {
```

Comment:
I think these tests test the AcidMetricsService more than Initiator, so they should be in TestAcidMetricsService

Reply:
Moved all the test to a common place TestCompactionMetrics.